backend-app/middlewares/api_version_controll.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response, NextFunction } from 'express';
+import { API_VERSION } from '@config/app_config';
+
+/**
+ * Middleware to set default API version in the request URL if not provided.
+ * If the API version is missing, it will be set to 'v3' by default.
+ * @param {Request} req - Express request object.
+ * @param {Response} res - Express response object.
+ * @param {NextFunction} next - Express next function to pass control to the next middleware or route handler.
```

Comment:
We dont need this. Cause we are now using ts.

Reply:
okay 👌